lib/readiness.js

```diff
@@ -125,26 +128,22 @@ async function _checkHttp({dependency} = {}) {
   return {ready, result};
 }
 
-async function _validateHttp({name, dependency} = {}) {
+async function _validateHttpGet({name, dependency} = {}) {
   try {
     assert.object(dependency, 'dependency');
     const {parameters} = dependency;
     assert.object(parameters, 'parameters');
-    const {url, method = 'GET', expectedStatus = 200} = parameters;
+    const {agent, url, headers, expectedStatus = 200} = parameters;
```

Comment:
Did you mean to put 'agent' in here? The config only works with plain JSON and 'agent' is a class instance.

Reply:
Good catch, removed, we do need to support use cases with httpAgent vs httpsAgent and various rejectUnauthroized settings that may vary from the bedrock default httpsAgent instance.  I think we can push this off to a future release.